src/app/qgisapp.cpp

```diff
@@ -1570,6 +1570,11 @@ QgisApp::QgisApp( QSplashScreen *splash, bool restorePlugins, bool skipVersionCh
   QShortcut *shortcutTracing = new QShortcut( QKeySequence( tr( "Ctrl+Shift+." ) ), this );
   connect( shortcutTracing, &QShortcut::activated, this, &QgisApp::toggleEventTracing );
 
+  QShortcut *shortcutToggleLinearCircularDigitizing = new QShortcut( QKeySequence( tr( "Ctrl+Shift+G" ) ), this );
```

Comment:
can  you explain the choice of "G" instead of something more predictable like "C"?

Reply:
I don't have good argument to explain this choice. Actually, I don't have idea about which one could be the best, "G" was the first letter available in the keyboard shortcuts key. Another esthetic reason is that "G" shape has linear and circulars strings... That is my only argument... So, of course, I am open for all suggestion. CTRL+Shift+C is already used for "Set CRS of Layer(s)".
And honestly "G" is not so easy to be associated with "Ctrl+Shift" with one hand...

Reply:
Just a reminder that plain "C" is hardcoded to Advanced digitizing *construction mode*

Reply:
My suggestion would be to use Ctrl+Shift+C ... the operation "Set CRS of Layer(s)" does not seem to be a common action that should have a predefined shortcut... what do others think?

Reply:
Why not, I am open to every other short cut, as long as that doesn't create issue... As this short cut can be changed in the QGis settings, I purpose to stay with this for now and to change this later when we will have the better "candidate"?